### ISSUE TITLE
ui-br-phone-number shows an error when the model starts with a value

### ISSUE THREAD

**Issue by ranpa** [assisrafael/angular-input-masks#301]
Hi,

I am using the _ui-br-phone-number_ for validating phone numbers in my app and realized that if I previously have a value in the ng-model (set on controller), the _ui-br-phone-number_ reports an error and invalidate my form. If I delete the last number and re-type it, the form turns to valid.

<img width="288" alt="screen shot 2017-09-20 at 5 58 33 pm" src="https://user-images.githubusercontent.com/3819511/30783684-8be8dcca-a11d-11e7-8a81-e6d90074170f.png">

`"angular-input-masks": "^2.6.0"`

```
"ionic": "3.6.0"
"type": "ionic1"
```

Tested on Google Chrome 61.0.3163.100

**Comment by assisrafael** [comment]
What is the angular version you are using?

**Comment by ranpa** [comment]
`Angular Version: 1.5.3`

I noticed that, if the ng-model value is set before the view is rendered (in a modal, for instance), the mask works fine. But, if the view is already displayed, and the ng-model is set after some action (like pressing a button), the error is shown and the form is invalid.

**Comment by assisrafael** [comment]
Can you write a test case that reproduce this scenario? Check the `src/br/phone/br-phone.test.js`

**Comment by ranpa** [comment]
I tried to write a test case, but, unfortunately, I am not that good on writing tests for angular (yet), so I was not able to pull request. 

However, I discovered the problem in my code: the api that I am using returns a masked phone number, and I was writing `vm.form.phone = data.phone` (in fact, the $modelValue was invalid). I solved that writing `vm.form.phone = data.phone.replace(/\D/g, '')` instead.

I don't know if this is the expected behaviour for angular-input-masks, but this is how it is working today.

Thanks for your attention!
